libcloud/test/dns/fixtures/cloudflare/nul.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Would make it easier if we could also automatically run tests on Windows, but Travis sadly doesn't support that (yet).